eb_hooks.py

```diff
@@ -1297,6 +1323,16 @@ def post_module_hook(self, *args, **kwargs):
         post_module_hook_zen4_gcccore1220(self, *args, **kwargs)
 
 
+def post_easyblock_hook(self, *args, **kwargs):
+    """Main post easyblock hook: trigger custom functions based on software name."""
+    if self.name in POST_EASYBLOCK_HOOKS:
+        POST_EASYBLOCK_HOOKS[self.name](self, *args, **kwargs)
+
+    # Always trigger this one for EESSI CVMFS/site installations, regardless of self.name
+    if os.getenv('EESSI_CVMFS_INSTALL') or os.getenv('EESSI_SITE_INSTALL'):
+        post_easyblock_hook_copy_easybuild_subdir(self, *args, **kwargs)
```

Comment:
```suggestion
if LooseVersion(EASYBUILD_VERSION) >= '5.1.1':
    def post_easyblock_hook(self, *args, **kwargs):
        """Main post easyblock hook: trigger custom functions based on software name."""
        if self.name in POST_EASYBLOCK_HOOKS:
            POST_EASYBLOCK_HOOKS[self.name](self, *args, **kwargs)

        # Always trigger this one for EESSI CVMFS/site installations, regardless of self.name
        if os.getenv('EESSI_CVMFS_INSTALL') or os.getenv('EESSI_SITE_INSTALL'):
            post_easyblock_hook_copy_easybuild_subdir(self, *args, **kwargs)
```
How about just version protecting the definition of the main hook?